Packs/ServiceNow/ReleaseNotes/2_2_12.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### ServiceNow v2
+- When querying a group with an ampersand (&) the query would break. Added a fix via a code flag to not parse ampersands.
```

Comment:
```suggestion
- Fixed an issue where the ***servicenow-query-groups*** command failed when the group name contained an ampersand (&).
```
```suggestion
- When querying a group with an ampersand (&) the query would break. Added a fix via a code flag to not parse ampersands.
```